### PR TITLE
modify r lang plugin (for lang-python{2,3})

### PR DESCRIPTION
**Pull request by Frefreak**
can't think of a better name for this new field

### CODE REVIEW

File: libr/lang/lang.c [hunk 1]
```diff
@@ -174,6 +174,8 @@ R_API RLangPlugin *r_lang_get_by_name (RLang *lang, const char *name) {
 		if (!strcasecmp (h->name, name)) {
 			return h;
 		}
+		if (h->trivial_name && !strcasecmp(h->trivial_name, name))
```
Comment (radare) [inline libr/lang/lang.c]: several issues here
* strcasecmp is not portable
* missing space before (
* missing braces around the if block

File: libr/util/lib.c [hunk 2]
```diff
@@ -11,7 +11,7 @@ R_LIB_VERSION(r_lib);
 
 #if __UNIX__
 #include <dlfcn.h>
-  #define DLOPEN(x)  dlopen(x, RTLD_GLOBAL | RTLD_NOW)
+  #define DLOPEN(x)  dlopen(x, RTLD_LOCAL | RTLD_NOW)
```
Comment (radare) [inline libr/util/lib.c]: i think taht it makes sense to use RTLD_LOCAL always, so that seems a good change

File: libr/lang/lang.c (hunk 1 above)
Comment (radare) [inline libr/lang/lang.c]: use r_str_casecmp()

File: libr/include/r_lang.h [hunk 3]
```diff
@@ -25,6 +25,7 @@ typedef struct r_lang_t {
 
 typedef struct r_lang_plugin_t {
 	const char *name;
+	const char *trivial_name;
```
Comment (radare) [inline libr/include/r_lang.h]: what about 'alias' instead of 'trivial_name' ?

File: libr/lang/lang.c (hunk 1 above)
Comment (radare) [inline libr/lang/lang.c]: in libr/util/str we have no alternative for this gnuism

File: libr/include/r_lang.h (hunk 3 above)
Comment (Frefreak) [inline libr/include/r_lang.h]: Yes, `alias` is much better.

**Pull request merged**